docs/how-to/package-with-hatch.md

```diff
@@ -0,0 +1,380 @@
+
+```toml
+[project]
+name = "textual-calculator"
+dynamic = ["version"]
+description = 'A example app'
+readme = "README.md"
+requires-python = ">=3.8"
+license = "MIT"
+keywords = []
+authors = [
+  { name = "Will McGugan", email = "redacted@textualize.io" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = []
+```
+
+We are interested in the `dependencies` value, which should list the app's dependencies.
+If you want a particular version of a project you can add `==` followed by the version.
+
+For the calculator, the only dependency is Textual.
+We can add Textual by modifying the following line:
+
+```toml
+dependencies = ["textual==0.47.1"]
+```
+
+At the time of writing, the latest Textual is `0.47.1`.
+The entry in `dependencies` will ensure we get that particular version, even when newer versions are released.
+
+See the Hatch docs for more information on specifying [dependencies](https://hatch.pypa.io/latest/config/dependency/).
+
+## Environments
+
+A common problem when working with Python code, is managing multiple projects with different dependencies.
+For instance, if we had another app that used version `0.40.0` of Textual, it *may* break if we installed version `0.47.1`.
+
+The standard way of solving this is with *virtual environments* (or *venvs*), which allow each project to have its own set of dependencies.
+Hatch can create virtual environments for us, and makes working with them very easy.
+
+To create a new virtual environment, navigate to the directory with the `pyproject.toml` file and run the following command (you will only need this on first run):
+
+```bash
+hatch env create
+```
+
+Then run the following command to active the virtual environment:
+
+```bash
+hatch shell
+```
+
+If you were to run `python` now, it will have our app and dependencies available for import:
+
+```
+$ python
+Python 3.11.1 (main, Jan  1 2023, 10:28:48) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
+Type "help", "copyright", "credits" or "license" for more information.
+>>> from textual_calculator import calculator
+```
+
+### Running the app
+
+You can launch the calculator from the command line with the following command:
+
+```bash
+python -m textual_calculator.calculator
+```
+
+The `-m` switch tells Python to import the module and run it.
+
+Although you can run your app this way (and it is fine for development), its not ideal for sharing.
+It would be preferable to have a single command to launch the app, so the user can easily run it from the command line.
```

Comment:
```suggestion
It would be preferable to have a dedicated command to launch the app, so the user can easily run it from the command line.
```

`python -m textual_calculator.calculator` _is_ a single command. An annoying one, but just one.